dask_cloudprovider/providers/gcp/tests/test_gcp.py

```diff
@@ -0,0 +1,72 @@
+import pytest
+
+import dask
+import googleapiclient.discovery
+from dask_cloudprovider.providers.gcp.instances import GCPCluster
+from dask.distributed import Client
+import dask.array as da
+from distributed.core import Status
+
+
+async def skip_without_credentials():
+    try:
+        googleapiclient.discovery.build("compute", "v1")
+    except:
+        pytest.skip(
+            """
+        You must configure your GCP credentials to run this test.
+
+            $ export GOOGLE_APPLICATION_CREDENTIALS=<path-to-gcp-json-credentials>
+
```

Comment:
I have configured credentials as described here but the test is still skipping. See the traceback I raised before.

I'm guessing something else is required to configure the credentials here, or the try/except is being greedy and catching some other exception.